src/oed.wppl

```diff
@@ -91,15 +95,14 @@ var EIG = function(args) {
                 var scoreY = m(x,y);
                 factor(scoreY);
             }
-            // TODO: just use AIG() here?
-            var mPosterior = inferM2(function() {
-                var m2Name = mNameSample(), m2 = mFuncs[m2Name];
-                var ll = m2(x,y);
-                factor(ll);
-                return m2Name
-            });
-            var kl = KL(mPosterior, mPrior);
-            return kl;
+
+            return AIG({
```

Comment:
Sure. Is this a decision i.e. "We're not going to call AIG as a subroutine in EIG" or a directive i.e. "Make AIG accept an optional model prior argument as an optimization"?